tsrc/gitlab.py

```diff
@@ -98,13 +98,20 @@ def get_project_id(self, project_name):
                 raise
 
     def find_opened_merge_request(self, project_id, source_branch):
-        url = "/projects/%s/merge_requests" % project_id
+        url = "/projects/%s/merge_requests?state=opened" % project_id
```

Comment:
this works, but

```python
self.make_request("GET", url, params={"state": "opened"})
```

feels better :)

Reply:
found in your code :P

fixing all of them